.github/workflows/fossa.yaml

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2021 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: fossa
+on:
+  repository_dispatch:
+    types: [fossa]
+  push:
+    branches:
+      - master
+      - release-*
+      - feature/*
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+      - release-*
+      - feature/*
+  workflow_dispatch: {}
+jobs:
+  fossa-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v3
+
+      - name: "Run FOSSA Scan"
+        uses: fossas/fossa-action@main # Use a specific version if locking is preferred
+        with:
+          api-key: f886d86744fc734c13de463b87eb3482 # This is a PUSH ONLY token that is safe to be shared with the public
```

Comment:
@berndverst we will need to change this with the central token that we get from CNCF once Mark helps us get one.

Reply:
Yes of course and additional we have to actually set up the different project in that central FOSSA org. But I doubt that will happen that soon.

Reply:
I am targeting v1.7. If we don't get a central account by then, I will create workflows across repos with my personal push token.